src/lib/viewers/doc/_docBase.scss

```diff
@@ -1,36 +1,55 @@
 @import '../../boxuiVariables';
 
+$thumbnail-border-radius: 4px;
+
 .bp {
     .bp-thumbnails-container {
         border-right: solid 1px $seesee;
         display: flex;
-        flex: 0 0 180px;
+        flex: 0 0 181px;
```

Comment:
why +1?

Reply:
To account for the 1px border on the right, so that the remaining width is actually 180

Reply:
Can you add a comment beside this, to clarify?